src/internal/Observable.ts

```diff
@@ -252,7 +252,7 @@ export class Observable<T> implements Subscribable<T> {
   /**
    * @method forEach
    * @param {Function} next a handler for each value emitted by the observable
-   * @param {PromiseConstructor} [promiseCtor] a constructor function used to instantiate the Promise
+   * @param {PromiseConstructorLike} [promiseCtor] a constructor function used to instantiate the Promise
```

Comment:
We can actually just remove these types in the comments altogether, as the documentation just gleans that information out of TypeScript.

Reply:
Done :+1: I only removed the types from the comment blocks I was touching. Happy to do the remainder in this file, just let me know .